packages/shared/src/types.ts

```diff
@@ -1,3 +1,8 @@
 export type Prettify<T> = {
   [K in keyof T]: T[K]
 } & {}
+
+export enum Network {
```

Comment:
is this still needed?

Reply:
I had already removed it locally, but only pushed the commit now.